site/docs/foundations/characteristics/index.mdx

```diff
@@ -0,0 +1,105 @@
+
+### What are characteristics?
+
+Characteristics are semantic groupings of visual attributes that are applied across relevant components. They drive consistency and support theming across complex libraries. They help to define the rules that style each component and aid the cohesiveness of the UI.
+
+### Why characteristics?
+
+From a design perspective, there is a need for aesthetic flexibility without decreasing the efficiency of build or the effectiveness of the components when implemented. From a dev perspective, it provides an easy way to reuse the same values, without the overhead of having to remember details such as hex values. Any time you want to tweak the value of a characteristic, you only have to do it in one place for it to reflect everywhere.
+
+Characteristics offer the following benefits:
+
+- Ability to theme the Salt inline with third-parties or CIB sub brand guidelines.
+- Increase consistency across all components, with the knock-on effect of improving the overall user experience e.g. reduced cognitive load.
+- Simplified design of new components and patterns e.g. drag-and-drop behavior.
+- Streamlined evolution – a single change to a characteristic will automatically cascade down through all related components within the library.
+
+### Token name convention
+
+When defining characteristic design tokens, they should generally be in the format `--salt-<characteristic name>-<variant>-<attribute>-<state>`
+
+1. Always begin with `--salt-`
+2. Follow this by the foundation name, e.g. `---salt-actionable-`, `---salt-text-`, `--salt-focused-`
+3. Append the variant if applicable, e.g. `--salt-actionable-cta-`, `--salt-text-h2-`
+4. Append the attribute, e.g. `--salt-actionable-cta-background`, `--salt-text-h2-fontSize`, `--salt-focused-outlineColor`
+5. Append the state if applicable, e.g. `--salt-actionable-cta-background-hover`
+
+### Styling with Characteristics
+
+Remember to wrap the characteristic you wish to use inside a `var()` function.
+
+For example, a Button is a component on which a user can perform an action, so you could use the following characteristics to style the component:
+
+```css
+.saltButton {
+  background: var(--salt-actionable-primary-background);
+  font-weight: var(--salt-actionable-fontWeight);
+}
+```
+
+Characteristics are both light/dark and density scoped, meaning that the value the characteristic points to will change according to the values given in the nearest `SaltProvider`. Considering the above, we would get that in light
+theme, `--salt-actionable-primary-background` resolves to `var(--salt-palette-interact-primary-background)`, which in light theme is `var(--salt-color-gray-60)` and in dark theme is `var(--salt-color-gray-300)`, of which foundation tokens then in turn resolve to their respective RGBA values.
+
+### Simple customization using characteristics
```

Comment:
I think we should actually delete this whole section until we agree as a team on how this is going to be dealt with (hopefully ASAP, and this will encourage the conversation)

e.g., --saltCard-borderRadius token used doesn't even exist anymore, since this doc was written a while ago, needs to be updated. we need designers to also give input on which attributes we want to expose, if we do go down the CSS API route
cc @bhoppers2008 @pseys

Reply:
yea for now i had just moved over the docs from storybook,, but lmk what all needs to be removed or changed

Reply:
just the bit from this heading to the next should be deleted, the rest is okay :) I made minor tweaks where it needed refreshing

Reply:
the stuff under "Simple customization using characteristics" ?